test/scripts/core_enable_disable.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Copyright © 2017-2018 The Crust Firmware Authors.
+# SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+
```

Comment:
You left out the last line of the header (see `Makefile`)

Reply:
Fixed.